.github/workflows/master_build_merge.yml

```diff
@@ -1307,6 +1323,14 @@ jobs:
               fi
             done
           }
+          zipDirectory() {
+            set -e
+            DIR="$1"
+            ARTIFACT_FILE="$2"
+            zip -r ${ARTIFACT_FILE} ${DIR}
+            set +e
```

Comment:
Remove `set -e` `set +e` if `mvn deploy` in line 1190 fails we won't be alerted. This is an internal function so not needed.